package.json

```diff
@@ -149,6 +149,7 @@
   "devDependencies": {
     "chai": "^4.2.0",
     "cross-env": "^5.2.0",
+    "eslint-plugin-react": "^7.11.1",
```

Comment:
I don't think `devDependencies` are installed when you `npm i aegir` - I think this has to be in `dependencies`